build-logic/documentation/src/main/groovy/gradlebuild/docs/GradleJavadocsPlugin.java

```diff
@@ -103,7 +113,7 @@ private void generateJavadocs(Project project, ProjectLayout layout, TaskContain
             // TODO: This breaks the provider
             task.setDestinationDir(generatedJavadocDirectory.get().getAsFile());
 
-            if (BuildEnvironment.INSTANCE.getJavaVersion().isJava11Compatible()) {
+            /*if (BuildEnvironment.INSTANCE.getJavaVersion().isJava11Compatible()) {
```

Comment:
❓ Same question as the other commented out code above

Reply:
I updated Javadocs to HTML5, so it's no longer needed. Deleting permanently.